migrate_coursetext_collection.ps1

```diff
@@ -0,0 +1,44 @@
+param(
+    [Parameter(Mandatory=$true)]
+    [string] $DevAccountKey,
+    [Parameter(Mandatory=$true)]
+    [string] $SitAccountKey,   
+    [Parameter(Mandatory=$true)]
+    [string] $PPAccountKey
+)
+
+$dtExe = "C:\ProgramData\chocolatey\lib\azure-documentdb-data-migration-tool\tools\dt.exe"
```

Comment:
Path to exe should be passed in as a param with a default value of the usual path